website/source/docs/job-specification/reschedule.html.md

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Add to JSON version as well

Reply:
Still need to update: https://www.nomadproject.io/api/json-jobs.html

Reply:
done